python/tests/test_aggregates.py

```diff
@@ -0,0 +1,214 @@
+import unittest
+from feldera import SQLContext
+from tests import TEST_CLIENT
+
+class TestAggregatesBase(unittest.TestCase):
+    def setUp(self) -> None:
+        self.data = [{"insert":{"id": 0, "c1": 1, "c2": 2, "c3": 3, "c4": 4, "c5": 5, "c6": 6, "c7": None, "c8": 8}},]       
```

Comment:
I was hoping that all the tests could use mostly the same data, but I guess this is good enough for now.
We can improve for the other aggregates.
Is this script being executed as part of the CI jobs?

Reply:
I'll keep that in mind for other aggregates. 

Python tests run in the CI. These tests are part of it.